recipes/commlib-py/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "commlib-py" %}
+{% set pypi_name = "commlib_py" %}
+{% set version = "0.11.1" %}
+{% set sha256 = "8c3782c371cb6e69d754fd79bccd7086dde36b4cd5560ce7f93fcc155c6c26e4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --use-pep517 -vvv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - poetry-core
+    - pip
+  run:
+    - hiredis >=2.1.1
+    - paho-mqtt >=1.6.1
+    - pika >=1.3.1
+    - pydantic >=1.10.4
+    - python >=3.8.0
+    - requests
+    - rich
+    - ujson >=5.7.0
+
+test:
+  imports:
+    - commlib
+  requires:
+    - coverage
+    - git
+  commands:
```

You could use the release tarball from https://github.com/robotics-4-all/commlib-py as the source above and include the tests with
```suggestion
  source_files:
    - test
  commands:
```

---

The issue was the release version discrepancy between Pypi and the repo release. There was a misunderstanding as to how to use the second source, it is not set with Pypi package (latest version) and tests from github as a second source